src/libexpr/function-trace.hh

```diff
@@ -7,18 +7,22 @@ namespace nix {
 
 struct FunctionCallTrace
 {
-    const Pos & pos;
+    string label;
 
-    FunctionCallTrace(const Pos & pos) : pos(pos) {
+    FunctionCallTrace(const string type, const Pos & pos) {
```

Comment:
This should be `const string & type`.

Reply:
oops, that was part of an ongoing experiment where I try to qualify the traces a bit better when the pos is undefined (like all the primop calls). Removed it from this PR for now.